_posts/2019-12-05-KotlinConf-2019.md

```diff
@@ -0,0 +1,1004 @@
+
+### 6. Deconstruct Myths
+Even with all this data, there are still some myths that exist concerning the language which you do not want to see spread around.
+Shelby and Katie broke down these myths in a [technical white paper](https://www.intuit.com/blog/uncategorized/kotlin-development-plan/){:target="_blank" rel="noopener noreferrer"} which they talk about on [Medium](https://medium.com/@srcohen1/kotlin-development-plan-5141087eaef0){:target="_blank" rel="noopener noreferrer"}.
+During the closing keynote of the conference, Maxim Shafirov (Jetbrains' CEO) also proposed to make an official white paper to help companies migrate.
+
+### 7. Can't stop, Won't Stop
+At this point, the team was on board and the train started going full-speed. 
+This is when they started a learning community to get other teams on board to spread the word.
+They would give presentations, do live coding and code reviews to help others get started too.
+
+### 8. Connect with Larger Community
+Kotlin has an amazing community, which allows you to grow even further.
+Katie mentions working on an Intuit [OpenSource project](https://github.com/intuit/truffle-shuffle){:target="_blank" rel="noopener noreferrer"}, attending conferences and [podcasts](https://talkingkotlin.com/){:target="_blank" rel="noopener noreferrer"}. 
+
+### Why Functional Programming
+Functional programming is a programming paradigm where computations are treated as mathematical functions.
+Shared state and mutable data are avoided which improves the quality of the codebase.
+It is also very helpful for multi-threaded workloads.
+Functional programming is easier in Kotlin because functions are so called [first-class](https://en.wikipedia.org/wiki/First-class_function){:target="_blank" rel="noopener noreferrer"} citizens.
+A good third party library which enhances the functional capabilities is [arrow-kt.io](https://arrow-kt.io){:target="_blank" rel="noopener noreferrer"}.
+It is similar to the [Vavr](https://www.vavr.io/) library for Java developers but with the benefits of the Kotlin idioms.
+
+<div class="responsive-video">
+    <iframe width="1164" height="655" src="https://www.youtube.com/embed/F2Der7xDNVM?start=10510" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+</div>
+<br/>
+
+## Your First Server with Ktor by [Big Nerd Ranch](https://twitter.com/bignerdranch){:target="_blank" rel="noopener noreferrer"}
+
+<span class="image left"><img class="p-image" style="max-width: 200px" alt="Big Nerd Ranch" src="/img/kotlinconf-2019/big-nerd-ranch.png"></span>
+
+This was a workshop on trying out [Ktor](https://ktor.io){:target="_blank" rel="noopener noreferrer"} and serves as a nice introduction to the framework.
+Ktor, backed by JetBrains, is a web framework for building asynchronous servers and clients in Kotlin.
```

Comment:
I have seen this already way in the beginning, maybe make it consistent on both lines

Reply:
Agreed. Leaving it out here.